demos/TransferLearning/TransferLearningDemo.py

```diff
@@ -0,0 +1,481 @@
+import warnings
+warnings.simplefilter('ignore')
+
+import ngraph_bridge
+ngraph_bridge.enable()
+#ngraph_bridge.set_backend('PLAIDML')
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+import os
+import random
+
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras.applications import ResNet50
+from tensorflow.keras.applications import MobileNetV2
+from tensorflow.keras.models import Sequential
+from tensorflow.python.keras import optimizers
+
+from keras.applications.resnet50 import preprocess_input
+from keras.preprocessing.image import ImageDataGenerator
+
+import ipywidgets as widgets
+from ipywidgets import VBox
```

Comment:
Just curious -- why import this one directly, and everything else as `widgets.Thing`?